alicloud/service_alicloud_polardb.go

```diff
@@ -0,0 +1,811 @@
+package alicloud
+
+import (
+	"encoding/json"
+	"strings"
+	"time"
+
+	"github.com/aliyun/alibaba-cloud-sdk-go/services/polardb"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/terraform-providers/terraform-provider-alicloud/alicloud/connectivity"
+)
+
+type PolarDBService struct {
+	client *connectivity.AliyunClient
+}
+
+func (s *PolarDBService) DescribePolarDBCluster(id string) (instance *polardb.DBCluster, err error) {
+	request := polardb.CreateDescribeDBClustersRequest()
+	request.RegionId = s.client.RegionId
+	dbClusterIds := []string{}
+	dbClusterIds = append(dbClusterIds, id)
+	request.DBClusterIds = id
+	raw, err := s.client.WithPolarDBClient(func(polarDBClient *polardb.Client) (interface{}, error) {
+		return polarDBClient.DescribeDBClusters(request)
+	})
+	if err != nil {
+		if NotFoundError(err) || IsExceptedErrors(err, []string{InvalidDBClusterIdNotFound, InvalidDBClusterNameNotFound}) {
```

Comment:
The API will not return a `NotFoundError` and it is built by provider. So, please change the line 29 to `if  IsExceptedErrors(err, []string{InvalidDBClusterIdNotFound, InvalidDBClusterNameNotFound}) {`